apps/web/components/dashboard/bookmarks/EditorCard.tsx

```diff
@@ -107,21 +108,6 @@ export default function EditorCard({ className }: { className?: string }) {
     setMultiUrlImportState({ urls, text });
   }
 
-  const onInput = (e: React.FormEvent<HTMLTextAreaElement>) => {
```

Comment:
Seems like you're basically reverting (https://github.com/hoarder-app/hoarder/pull/247)? Is that intentional?

Reply:
ok, misunderstood what resize was meant. reverted that part